cmd/mr_create.go

```diff
@@ -111,6 +112,22 @@ func runMRCreate(cmd *cobra.Command, args []string) {
 	if err != nil {
 		log.Fatal(err)
 	}
+
+	ofilename, err := cmd.Flags().GetString("file-open-editor")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if ofilename != "" && filename != "" {
+		log.Fatalf("Cannot specify both -F and -o options.")
```

Comment:
I think you meant `-f` instead of `-o` correct? :))

Reply:
Fixed in new version :)